README.md

```diff
@@ -9,7 +9,7 @@ The Outbound Funds app is a community developed and maintained Open Source Commo
 Links:
 
 -   [Documentation/Wiki](https://github.com/SalesforceFoundation/OutboundFundsModule/wiki)
--   [Power of Us Hub group](https://powerofus.force.com/s/group/0F980000000CvlMCAS/community-project-outbound-funds)
+-   [Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE/community-project-outbound-funds)
```

Comment:
```suggestion
-   [Outbound Funds group on Trailblazer Community](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHi0SAE)
```

Reply:
I caught this as well and pushed a change a little while ago. Resolving this comment.